doc/stdlib/linalg.rst

```diff
@@ -638,17 +638,45 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
 .. function:: expm(A)
 
-   Compute the matrix exponential of ``A``.
+   Compute the matrix exponential of ``A``, defined by
+
+         .. math::
+
+            e^A = \sum_{n=0}^{\inf} \dfrac{A^n}{n!}.
+
+   For symmetric or Hermitian ``A``, an eigendecomposition (:func:`eigfact`) is used, otherwise the scaling and squaring algorithm (see [H05]_) is chosen.
+
+   .. [H05] Nicholas J. Higham, "The squaring and scaling method for the matrix
+      exponential revisited", SIAM Journal on Matrix Analysis and Applications,
+      26(4), 2005, 1179-1193.
+      `doi:/10.1137/090768539 <http://dx.doi.org/10.1137/090768539>`
 
 .. function:: logm(A)
 
-   Compute the matrix logarithm of ``A``.
+   If ``A`` has no negative real eigenvalue, compute the principal matrix logarithm of ``A``, i.e. the unique matrix :math:`X` such that :math:`e^X = A` and :math:`-\pi < Im(\lambda) < \pi` for all the eigenvalues :math:`\lambda` of :math:`X`. If ``A`` has nonpositive eigenvalues, a warning is printed and whenever possible a nonprimary matrix function is returned.
```

Comment:
@andreasnoack You're right, thank you for spotting this, I'll fix it and submit a separate PR just for the documentation (I'll have to adapt it to #11943).